content/resources/exporting-python-projects.md

```diff
@@ -0,0 +1,56 @@
+# Packaging and Publishing Python Libraries
```

Comment:
Front matter data is needed for every guide! Please add, check this out for ref: https://github.com/DSACMS/ospo-guide/blob/main/content/resources/github101.md?plain=1#L1-L15